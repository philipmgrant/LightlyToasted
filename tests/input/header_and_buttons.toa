|Header: TestHead
|ButtonTarget1: https://1.example.com
|ButtonLabel1: example1
|ButtonTarget2: https://1.example.com
|ButtonLabel2: example2
|DismissButton
Testing header
and
buttons
