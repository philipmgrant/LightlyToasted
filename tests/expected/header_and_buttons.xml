<toast><visual><binding template="ToastImageAndText04"><image id="1" src=""/><text id="1">Testing header</text><text id="2">and</text><text id="3">buttons</text></binding></visual><header id="TestHead" title="TestHead" arguments=""/><actions><action arguments="https://1.example.com" activationType="protocol" content="example1"/><action arguments="https://1.example.com" activationType="protocol" content="example2"/><action arguments="dismiss" activationType="system" content="Dismiss"/></actions></toast>